### ISSUE TITLE
[WASM] `ScrollBar` isn't going to appear while dynamic content is being aggregated 

### ISSUE THREAD

**Issue by sigmarsson** [unoplatform/uno#6077]
## Scrollbar isn't showing up in the scroll viewer

```
        <nowinui:ScrollViewer Grid.Row="1"
                              VerticalScrollMode="Auto"
                              VerticalScrollBarVisibility="Auto">
            <wasmcc:WebAssemblyHtmlControl HtmlContent="{x:Bind HtmlContent, Mode=OneWay}" />
        </nowinui:ScrollViewer>
```

`HtmlContent` is aggregating `<div>` elements in the view model. 

## Expected behavior

Vertical scroll bar appears as property `HtmlContent` is aggregating.

## Environment

Nuget Package:

  <ItemGroup>
    <PackageReference Include="Microsoft.Extensions.Logging" Version="5.0.0" />
    <PackageReference Include="Microsoft.Windows.Compatibility" Version="5.0.2" />
    <PackageReference Include="Ninject" Version="3.3.4" />
    <PackageReference Include="Uno.Extensions.Logging.WebAssembly.Console" Version="1.1.0-dev.1" />
    <PackageReference Include="Uno.WinUI.WebAssembly" Version="3.8.0-dev.225" />
    <PackageReference Include="Uno.WinUI.RemoteControl" Version="3.8.0-dev.225" Condition="'$(Configuration)'=='Debug'" />
    <PackageReference Include="Uno.Wasm.Bootstrap" Version="2.1.0" />
    <PackageReference Include="Uno.Wasm.Bootstrap.DevServer" Version="2.1.0" />
  </ItemGroup>

- [ ] Uno.UI / Uno.UI.WebAssembly / Uno.UI.Skia
- [x] Uno.WinUI / Uno.WinUI.WebAssembly / Uno.WinUI.Skia
- [ ] Uno.SourceGenerationTasks
- [ ] Uno.UI.RemoteControl / Uno.WinUI.RemoteControl
- [ ] Other: 

Nuget Package Version(s):

Affected platform(s):

- [ ] iOS
- [ ] Android
- [x] WebAssembly
- [ ] WebAssembly renderers for Xamarin.Forms
- [ ] macOS
- [ ] Skia
  - [ ] WPF
  - [ ] GTK (Linux)
  - [ ] Tizen
- [ ] Windows
- [ ] Build tasks
- [ ] Solution Templates

IDE:

- [ ] Visual Studio 2017 (version: )
- [ ] Visual Studio 2019 (version: )
- [ ] Visual Studio for Mac (version: )
- [ ] Rider Windows (version: )
- [ ] Rider macOS (version: )
- [ ] Visual Studio Code (version: )

Relevant plugins:

- [ ] Resharper (version: )

**Comment by jeromelaban** [comment]
Thank you for the report. Can you provide a full repro sample attached to the issue ?

**Comment by sigmarsson** [comment]
@jeromelaban yes, please consider the respective commit [here](https://github.com/sigmarsson/6051).

**Comment by dr1rrb** [comment]
Direct link to the commit : https://github.com/sigmarsson/6051/commit/29ba9d2c8459e854978bdbd46123d2786832e3f7

**Comment by MartinZikmund** [comment]
This is only relevant to native wasm, closing for now
